dependencies.repos

```diff
@@ -10,6 +10,10 @@
     local-name: fcl
     uri: https://github.com/flexible-collision-library/fcl.git
     version: 0.6.1
+- git:
+    local-name: hpp-fcl
+    uri: https://github.com/humanoid-path-planner/hpp-fcl.git
+    version: v2.4.1
```

Comment:
Can we update this to point to the latest hash to see if the unit tests will pass?

Reply:
Yes, but I wanted to test this locally first.

Reply:
I've done more extensive testing, see the table in the first post. I've also updated the PR to use the `devel` branch.